broadinstitute/pearl
# [JN-535] edit notification configs

This enables some properties of notification configs to be editable via UI.  Not the email templates, but the param values like max number of reminders.  (The backend supports creating a new email template, but updating the frontend was outside the scope of this ticket).  This also upgrades the notifcation config UX to match the recently restyled populate UX.

![image](https://github.com/broadinstitute/juniper/assets/2800795/2fd3869d-0be6-460e-b620-f97481f5b729)

TO TEST:
1. go to http://localhost:3000/ourhealth/studies/ourheart/env/sandbox/notificationContent/configs
2. choose a config
3. update the Max reminders value
4. hit save
5. confirm the updated value is saved, and persists on refresh
